db/config.hh

```diff
@@ -500,6 +500,8 @@ public:
     named_value<bool> enable_tablets;
     named_value<uint32_t> view_flow_control_delay_limit_in_ms;
 
+    bool create_service_levels_table = true; // Some upgrade tests don't want enterprise-only tables
```

Comment:
I think you are right.

Actually, this flag is not used anywhere - but only because I forgot to migrate the only part of the code which set it to false (in schema_change_test.cc). The test passed despite that; I was wondering why (new columns were added to system_distributed_keyspace.service_levels table) but it looks like system distributed tables are not taken into account by the test since e26aa836a97b5921e3890d35fa9111fdc75d246f.

I'll drop the flag because it is not needed for anything anymore.